### DISCUSSION TITLE
Network issues recently?

### DISCUSSION THREAD

**Discussion by MattCWheeler** [community/community#45288] [Actions]
### Select Topic Area

Bug

### Body

Had anyone else gotten a really bad run of network timeouts from inside GH Actions in the last few days? I've seen everything from TCP handshake failures trying to auth with ghcr.io, to 400 errors trying to pull from assorted public repos (inside of GH Actions), network timeouts communicating with AWS STS, etc. May have started Friday, it's _certainly_ been very bad yesterday and somewhat today.

**Comment by anthonator** [comment]
I’ve recently started seeing jobs queue indefinitely. I have 3 that have been queued for over an hour. All waiting for a runner. Maybe not network related but definitely seeing weird behavior.

**Comment by leonschoorl** [comment]
We've also seeing a some CI failures with errors pulling docker images from ghcr.io lately, most often:
```
ERROR: Job failed: failed to pull image "ghcr.io/[..]" with specified policies [always]: Error response from daemon: Head "https://ghcr.io/v2/[...]": dial tcp 140.82.121.34:443: i/o timeout (manager.go:250:30s)
```
